docker/llm/finetune/lora/docker/bigdl-lora-finetuing-tdx-entrypoint.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
what is the difference between two entrypoints? Can be only one entrypoint file here?